docs/get-started/cardano-node/new-tracing-system/_category_.yml

```
@@ -0,0 +1,4 @@
+position: 6 # float position is supported
+label: 'New tracing system'
```

```suggestion
label: 'Tracing system'
```
This change is debatable of course since the new tracing system is literally called the "new tracing system"... but from a documentation point of view the word "New" in the sidebar is going to look incongruous in the near future and might even be misleading when it is less "new" than new arrivals to the Cardano community.

It would be accurate both now _and_ in the future by simply titling it _Tracing system_ especially since the top of that page explains the change & the terminology being used.  The shorter title would still be accurate even if the tracing system develops a more proper versioning scheme someday.

---

I agree with @rphair on this. Probably not a blocker for the PR but would need to be addressed soonish.

---

@disasm, should we implement versioning for the tracing system. So that new = V2.0